src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35612/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:35612/project/issues?id=sample&branch=feature3&issues=AWFQsgNf5tw7YoN-8foC&open=AWFQsgNf5tw7YoN-8foC)

[ISSUE_KEY]: AWFQsgNf5tw7YoN-8foC